GetandTake/Pages/Categories/Edit.cshtml

```diff
@@ -1,5 +1,16 @@
 ﻿@page
 @model GetandTake.Pages.Categories.EditModel
+@{
+    ViewData["Title"] = "Edit Category";
+}
+@if (!ModelState.IsValid)
```

Comment:
C# contexts should be also separated by empty lines - it simplifies reading. Don't forget to apply it for all sections :)